frontend/src/containers/LoginPage/LoginPage.js

```diff
@@ -2,32 +2,97 @@ import React from 'react';
 import { Grid, Hidden } from '@material-ui/core';
 import LoginForm from '../../components/LoginForm/LoginForm';
 import sideImage from '../../assets/LoginPage/side-image.svg';
+import { authenticate, getCurrentUser } from '../../api/PublicAPI';
+import CustomSnackbar from '../../components/CustomSnackbar/CustomSnackbar';
 import './styles.css';
 
 class LoginPage extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      showWrongCredentialsError: false,
+      showGeneralError: false
+    }
+  }
+
+  handleLogin = (mail, password) => {
+    authenticate(mail, password).then(responseToken => {
+
+      const token = responseToken.data;
+      getCurrentUser(token).then(responseUserData => {
+
+        const { id, firstname, lastname, email } = responseUserData.data;
```

Comment:
change to firstName, lastName everywhere pls. This is what I get when you use incorrect capitalization 
![image](https://user-images.githubusercontent.com/37108355/116817817-d04ace80-ab57-11eb-94d9-fd07cc2563c0.png)